backend/src/db/migrations/20250324142105_secret-rotation-v2.ts

```diff
@@ -0,0 +1,57 @@
+import { Knex } from "knex";
```

Comment:
-- ignore position of comment --

What if I don't want to map the generated secrets to Infisical secrets? We already have a dedicated endpoint for fetching the active and retired generated credentials. We no longer have to stick to our secret rotation v1 convention of outputting generated credentials as Infisical secrets